fuzzing/private/binary.bzl

```diff
@@ -36,6 +37,7 @@ Provider for storing information about a fuzz test binary.
         "corpus_dir": "The directory of the corpus files used as input seeds.",
         "dictionary_file": "The dictionary file to use in fuzzing runs.",
         "engine_info": "The `FuzzingEngineInfo` provider of the fuzzing engine used in the fuzz test.",
+        "options_file": "A file containing fuzzing engine and sanitizer options to use during execution. The file loosely follows the INI format and currently only applies to OSS-Fuzz.",
```

Comment:
The line above exceeds the usual line length limit, so I kept this as one line as well. Should I wrap both instead?

Reply:
Sure, wrapping both makes sense to me.

Reply:
Done.